ros2pkg/ros2pkg/api/create.py

```diff
@@ -17,6 +17,12 @@
 import sys
 
 import em
+try:
+    from em import Configuration
+    em_has_configuration = True
+except ImportError:
+    em_has_configuration = False
```

Comment:
This looks totally reasonable to me (though I'm the one who suggested it, so that's not surprising).

Before approving this, I'd like to get @cottsay 's take on this pattern.